PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-lib/src/main/java/com/microsoft/azure/toolkit/intellij/common/AzureIntegerInput.java

```diff
@@ -24,11 +24,19 @@
     @Setter
     private Integer maxValue;
 
+    public AzureIntegerInput() {
+        super();
+        this.setDefaultValue(0);
+    }
+
     @Nullable
     @Override
     public Integer getValue() {
         final String text = getText();
-        if (StringUtils.isBlank(text) || !StringUtils.isNumeric(text)) {
+        if (StringUtils.isBlank(text)) {
```

Comment:
do we also need to check isRequired here?

Reply:
no need. validation will do the check.